src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -56,9 +62,9 @@ export const TodoFilter: React.FC<Props> = ({ todos, setFilter }) => {
             data-cy="statusSelect"
             onChange={(e) => setOption(e.target.value as FilterOption)}
           >
-            <option value={FilterOption.All}>All</option>
-            <option value={FilterOption.Active}>Active</option>
-            <option value={FilterOption.Completed}>Completed</option>
+            {dropList.map(optionList => (
+              <option value={optionList.value}>{optionList.label}</option>
```

Comment:
you missed `key` attribute

Reply:
sure, I have added key to the droplist option.